flaskr/algorithm.py

```diff
@@ -61,18 +67,99 @@ def generate_profile(portfolio, finnhub_client):
             'SENTIM': S,
             'NUM': int(num)
         })
+
+        totalValueShares = totalValueShares + (price * int(num))
+
+        # Assign sector weight based on number of shares owned in sector
+        # Currently finnhub is returning random words for most of the fields of this
+        # call, including finnhubIndustry. Until this is fixed, the interests factor
+        # will not work correctly
```

Comment:
I'm not sure if @elrodrigues mentioned this already or not but I tested this out in Postman and it looks like we're getting random words from the finnhub client because we're currently using the sandbox API key in our Flask app. If we make requests using the regular API key then it should work fine. I'm assuming that we'll switch keys sometime down the line as we move on to testing our final application.